cmd/config_git_set.go

```diff
@@ -104,6 +129,8 @@ func (c configGitSetConfig) Prompt(f fn.Function) (configGitSetConfig, error) {
 		return c, err
 	}
 
+	fmt.Printf("%#v\n", c.metadata)
```

Comment:
This might be a leftover debug statement?

Reply:
You are right, good catch! Thanks